project/server/app.js

```diff
@@ -1,26 +1,109 @@
 const express = require('express');
 const bodyParser = require('body-parser');
+const cors = require('cors');
+const IsValid = require('../src/components/validationComponent');
+const mongoose = require('mongoose');
+const MongoClient = require('mongodb');
+const Schema = mongoose.Schema;
+const users = require('./routes/users');
+const router = express.Router();
+const Users = require('./models/Users.js');
+const multer = require('multer');
+const assert = require('assert');
 
-const db = require('./utils/dataBaseUtils.js');
+const db = mongoose.connection;
+mongoose.connect('mongodb://localhost:27017/myDataBase', ((err)=>{
```

Comment:
store this url in the file with constants.
Generally, store all URLs in your application inside constants file